src/strings/en-gb.json

```diff
@@ -597,6 +597,8 @@
     "PreferredNotRequired": "Preferred, but not required",
     "PreferEmbeddedTitlesOverFileNamesHelp": "This determines the default display title when no internet metadata or local metadata is available.",
     "PreferEmbeddedTitlesOverFileNames": "Prefer embedded titles over filenames",
+    "PreferEmbeddedEpisodeInfosOverFileNamesHelp": "This uses the episode informations from the embedded metadata if available.",
+    "PreferEmbeddedEpisodeInfosOverFileNames": "Prefer embedded episode informations over filenames",
```

Comment:
If en-gb version doesn't differ from en-us you don't need to specify it, only specify translation parts which differ.

Reply:
Well we already have so many duplicate strings in the British translation I don't think it matters at this point. I assume the same thing happens for all similar languages.

Reply:
That doesn't mean we should keep piling dupes tho ;)